src/components/Items/ProfileContact.js

```diff
@@ -14,46 +18,75 @@ const ProfileContact = ({
   showArrow,
   userId,
   item,
-  ImageComponent = null
-}) => (
-  <Pressable
-    onPress={onPress}
-    android_ripple={{
-      color: COLORS.gray1,
-      borderless: false,
-      borderRadius: 10
-    }}
-    style={styles.pressable}>
-    <View style={styles.container}>
-      <View style={styles.profile}>
-        {ImageComponent || (
-          <Image
-            testID="image"
-            style={styles.image}
-            source={{uri: photo !== '' ? photo : undefined}}
-          />
-        )}
-        <Text style={styles.fullname}>{fullname}</Text>
-      </View>
-      {showArrow && (
-        <>
-          {userId !== item.user_id && (
-            <View>
-              <Mi name="arrow-forward-ios" size={18} />
-            </View>
+  ImageComponent = null,
+  disabled = false,
+  from
+}) => {
+  const handleYouText = () => {
+    if (!from && (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER)) {
+      return '(You)';
+    }
+    return '';
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `handleYouText` has been introduced which checks if the user is anonymous or not and returns a string accordingly. This function seems to be used later in the component rendering. There's no apparent issue with this change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->